easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -919,9 +926,7 @@ def _generate_template_values(self, ignore=None, skip_lower=True):
         # (eg the run_setp code in EasyBlock)
 
         # step 1-3 work with easyconfig.templates constants
-        # use a copy to make sure the original is not touched/modified
-        template_values = template_constant_dict(copy.deepcopy(self._config),
-                                                 ignore=ignore, skip_lower=skip_lower)
+        template_values = template_constant_dict(self._config, ignore=ignore, skip_lower=skip_lower)
```

Comment:
no, since `template_constant_dict` only uses the 1st argument read-only

this was a better-safe-than-sorry move, which is not required and turns out to be quite expensive, especially for the `easyconfigs` test suite